src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -249,4 +251,51 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
             return new SchemaShape(resultDic.Values);
         }
     }
+
+    /// <summary>
+    /// Extension methods for the static-pipeline over <see cref="PipelineColumn"/> objects.
+    /// </summary>
+    public static class IidChangePointStaticExtensions
```

Comment:
>IidChangePointStaticExtensions [](start = 24, length = 30)

Can you please create TimeSeriesStatic.cs file and put all extensions into that file?